macaroonbakery/bakery.py

```diff
@@ -0,0 +1,208 @@
+# Copyright 2017 Canonical Ltd.
+# Licensed under the LGPLv3, see LICENCE file for details.
+
+import base64
+try:
```

Comment:
All of these are in [six](http://pythonhosted.org/six/#module-six.moves), so we could just do `from six.moves.http_cookiejar import Cookie, CookieJar` with no try block. This is mostly a mechanical change, though, and could probably be deferred to a follow-up branch.